include/wx/osx/cocoa/private.h

```diff
@@ -45,6 +45,7 @@ void WXDLLIMPEXP_CORE wxOSXSetImageSize(WX_NSImage image, CGFloat width, CGFloat
 wxBitmap WXDLLIMPEXP_CORE wxOSXCreateSystemBitmap(const wxString& id, const wxString &client, const wxSize& size);
 WXWindow WXDLLIMPEXP_CORE wxOSXGetMainWindow();
 WXWindow WXDLLIMPEXP_CORE wxOSXGetKeyWindow();
+WXImage WXDLLIMPEXP_CORE wxOSXGetNSImageFromNSCursor(const WXHCURSOR cursor);
```

Comment:
I'm not sure if we really need this function, it is really trivial and IMO the code would be more clear if it were just inlined in the only place where it is used.

Reply:
we need to do it that way, because the implementation is objective-c, and can have it's implementation only in a .mm file, therefore I've asked Igor to expose it that way